docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Dockerfile for FermiLib+ProjectQ
```

Comment:
I wonder, do we also want this docker image to be setup with psi4 (and/or pyscf)? Because I feel that installing those things is a large part of the challenge that we would want to help people avoid by providing the docker image. @jarrodmcc do you have thoughts about that?